README.md

```diff
@@ -58,179 +58,110 @@ For more information on the importance of a professional README for your reposit
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Swing & Notes<br>Restaurante de Jazz en Vivo] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+**[Swing & Notes<br>Restaurante de Jazz en Vivo]** is the web page of Swing & Notes, a restaurant in Bogotá with jazz live music. This is a capstone project that required me to distribute the layout with HTML, style it, and make it responsive for mobile and desktop with CSS.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://www.w3.org/">HTML</a></li>
+    <li><a href="https://www.css3.com/">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://nodejs.org/es">Nodejs</a></li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://www.github.com">Express.js</a></li>
```

Comment:
- [ ] As the [previous reviewer suggested](https://github.com/danielf-canon/Capstone-project/pull/1#discussion_r1289233125), please make sure that all pages are redirected to the correct associated website. You did well fixing the other links 👍👍 kindly rectify this as well ✨ The link is redirecting to the `GitHub`, however, the text written is `Epress.js`. 

  _You can safely remove a stack if it is not applicable to your current project_